source/Halibut.Tests/Support/TestAttributes/LatestClientAndLatestServiceTestCasesAttribute.cs

```diff
@@ -48,11 +48,18 @@ public static IEnumerable GetEnumerator(bool testWebSocket, bool testNetworkCond
                 {
                     serviceConnectionTypes.Remove(ServiceConnectionType.Polling);
                 }
+                
+                ForceProxyClientType[] proxyClientTypesToTest = new ForceProxyClientType[0];
+                if (testAsyncAndSyncClients)
+                {
+                    proxyClientTypesToTest = ForceProxyClientTypeValues.Values;
```

Comment:
We have an existing pattern e.g. ServiceConnection`Types.All` - should we keep these copnsistent